src/dataServices/dynamoDbDataService.ts

```diff
@@ -325,6 +310,30 @@ export class DynamoDbDataService implements Persistence, BulkDataAccess {
         return getExportStatusResponse;
     }
 
+    buildExportJob(initiateExportRequest: InitiateExportRequest): BulkExportJob {

```

Comment:
Why make this a class method?

Reply:
The method uses class variable `this.enableMultiTenancy`. Although it is the same as checking if tenantId exist, I think it's more readable using `this.enableMultiTenancy`.